package.json

```diff
@@ -11,16 +11,30 @@
         "zerorpc": "latest"
     },
     "devDependencies": {
+        "@types/chai": "^4.1.5",
+        "@types/chai-as-promised": "^7.1.0",
+        "@types/datatables.net": "^1.10.13",
+        "@types/jquery": "^3.3.6",
+        "@types/jquery.contextmenu": "^1.7.33",
+        "@types/mocha": "^5.2.5",
+        "@types/node": "^8.9.2",
+        "@types/tail": "^1.2.0",
         "chai": "^4.1.2",
         "chai-as-promised": "^7.1.1",
-        "electron": "1.8.4",
+        "electron": "^1.8.4",
         "electron-packager": "^8.7.2",
         "electron-rebuild": "^1.5.7",
-        "mocha": "^5.1.1",
-        "spectron": "^3.8.0"
+        "mocha": "^5.2.0",
+        "rimraf": "^2.6.2",
+        "spectron": "^4.0.0",
+        "ts-node": "^7.0.1",
+        "tslint": "^5.11.0",
+        "typescript": "^3.0.3"
     },
     "scripts": {
-        "test": "mocha ui/tests/",
-        "start": "electron ."
+        "build": "rimraf ui/dist && tsc",
```

Comment:
Why rimraf and not just simply `rm -rf`. What is the advantage? I see the package but it does not explain why: https://github.com/isaacs/rimraf

Reply:
My guess, and my reason for not using `rm -rf` is that it would not work on Windows, unless you had some extra stuff installed.